.dockerignore

```diff
@@ -11,10 +11,14 @@ web-api/.dynamodb
 web-api/.nyc_output
 web-api/.serverless
 web-api/coverage
-web-api/.cache
```

Comment:
Because GitHub lacks effective pull-request-level feedback mechanisms that must be addressed before a pull request can be merged, I am using this file as the place to attach PR-level comments.